go/kms/README.md

```diff
@@ -0,0 +1,96 @@
+# AWS SDK for Go code examples for AWS Key Management Service (AWS KMS)
```

Comment:
...for **AWS KMS**
(short form in title)

Reply:
Fixed